### ISSUE TITLE
怎么控制多section多cell 显示的数量的,我现在一个tableview 第一段中第一个cell和第二个cell不一样,第二段中只有header没有cell,现在加载不出对应的骨架屏,一整屏都是第一个cell 的样式.着急使用,谢谢您

### ISSUE THREAD

**Issue by 15949893621** [tigerAndBull/TABAnimated#133]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
阅读文档，谢谢



发自我的iPhone


------------------ 原始邮件 ------------------

**Comment by 15949893621** [comment]
您好 你说的我都看了,我是用swift混编开发的,下面2种方法都试过了都不可以,我是用2.5.1 版本的

//方法使用1
        tableView.tabAnimated = TABTableAnimated(cellClassArray: [SFFindCommunityBannerTableViewCell.self,
                                                                  SFFindCommunityBrandsAndSpecialActivitiesTableViewCell.self],
                                                 cellHeightArray: [NSNumber(130),NSNumber(130)],
                                                 animatedCount: [NSNumber(1),NSNumber(1)])
        
//方法使用2
//        tableView.tabAnimated = TABTableAnimated(cellClassArray: [SFFindCommunityBannerTableViewCell.self,
//                                                                  SFFindCommunityBrandsAndSpecialActivitiesTableViewCell.self],
//                                                 cellHeightArray: [NSNumber(130),NSNumber(130)], animatedCount: [NSNumber(1),NSNumber(1)], animatedSectionArray: [NSNumber(1),NSNumber(1)])

**Comment by tigerAndBull** [comment]
仔细阅读接口使用说明

**Comment by tigerAndBull** [comment]
你需要确定你的视图结构是row为主体，还是section为主体

**Comment by tigerAndBull** [comment]
该issue已关闭，如果你仍存在问题，提供具体信息再次将其打开即可。
